pkg/controller/common.go

```diff
@@ -493,6 +493,24 @@ func (c *SharedState) addPod(pod *v1.Pod, updateOption updateWithLock) {
 	}
 
 	for _, volume := range pod.Spec.Volumes {
+		// TODO: investigate if we need special support for CSI ephemeral volume or generic ephemeral volume
```

Comment:
Please file an issue for this.

Reply:
Done 👍 
https://github.com/abhisheksinghbaghel/azuredisk-csi-driver/issues/383